spec/controllers/gift_requests_controller_spec.rb

```diff
@@ -46,6 +46,15 @@
 
       expect(assigns(:gift_request).errors.full_messages).to_not be_empty
     end
+
+
+    it "restricts non therapists from doing a create action" do
+      login_as_admin
+
+      post :create, gift_request: {}, id: @list.id
+      expect(response).to redirect_to(root_path)
+      expect(assigns[:gift_request]).to be_nil
```

Comment:
this can just test something like `expect(GiftRequest.all).to match_array([])` instead - i'm not sure what this line is actually doing